examples/asset_licences.md

```diff
@@ -0,0 +1,9 @@
+The "Tuffy" font is public domain, was made by Thatcher Ulrich, and was accessed from here: http://www.publicdomainfiles.com/show_file.php?id=13486218041168
+The loop doodle_pop.ogg is public domain and was made by MrZeusTheCoder.
+opengl/resources/background.png is public domain, was made by Nidhoggn, and was accessed from here: https://opengameart.org/content/backgrounds-3
+shader/resources/background.jpg is public domain, was taken by Arcana Dea, and was accessed from here: https://www.publicdomainpictures.net/en/view-image.php?image=10979&picture=monarch-butterfly
```

Comment:
Maybe play around a bit with the compression or size, so we don't have a 320% file size increase